File-Formats/DocIO/word-to-pdf.md

```diff
@@ -3427,4 +3427,8 @@ Partially supported. At present, the text in a line is scaled uniformly to match
 * [Is it possible to perform Word to PDF conversion in Azure Environment ?](https://www.syncfusion.com/kb/7751/is-it-possible-to-perform-word-to-pdf-conversion-in-azure-environment)
 * [How to convert Word document to PDF in AWS Lambda](https://www.syncfusion.com/kb/11905/how-to-convert-word-document-to-pdf-in-aws-lambda)
 * [How to convert Word document to PDF in Azure App service on Linux](https://www.syncfusion.com/kb/11888/how-to-convert-word-document-to-pdf-in-azure-app-service-on-linux)
-* [How to mail merge Word documents and convert to PDF in Azure Functions v2](https://www.syncfusion.com/kb/11197/how-to-mail-merge-word-documents-and-convert-to-pdf-in-azure-functions-v2)
\ No newline at end of file
+* [How to mail merge Word documents and convert to PDF in Azure Functions v2](https://www.syncfusion.com/kb/11197/how-to-mail-merge-word-documents-and-convert-to-pdf-in-azure-functions-v2)
```

Comment:
Order the KB title as per platforms

1. Word to PDF related problems and solutions (font, emf)
2. UWP related KBs
3. Linux
4. Azure App service
5. Azure functions
6. AWS
7.